usr/lib/zabbix-check/bin/zabbix-check-sshkey-discovery

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env sh
+
+set -e
+
+return_val="{\"data\":["
+
+# reading home paths of users from /etc/passwd and checks
+# if there is an authorized_keys file
+while read -r line; do
+    auth_keyfile_path="$(echo "${line}" | cut -d":" -f6)/.ssh/authorized_keys"
```

Comment:
There may also be a `~/.ssh/authorized_keys2`, we should check that too. The file is not really well documented but here (https://www.debian.org/security/key-rollover/index.de.html) you can see that it is also used under Debian.

Reply:
done